htdocs/compta/prelevement/fiche-rejet.php

```diff
@@ -97,7 +97,8 @@
 
 if ($id > 0 || $ref) {
 	if ($object->fetch($id, $ref) >= 0) {
-		$head = prelevement_prepare_head($object);
+		$salaryBonPl = $object->checkIfSalaryBonPrelevement($id);
```

Comment:
This method can't be find.
I suggest to create a PR to add this method. Adding a method  does not depend on other code so can be done into an atomic isolated PR.

Reply:
here, unfortunately, it is essential in the prepare_head function, so I think I'll leave it.